Packs/CyberTotal/Integrations/CyberTotal/CyberTotal.yml

```
@@ -0,0 +1,845 @@
+        malicious.
+      type: String
+    - contextPath: URL.Malicious.Vendor
+      description: The vendor reporting the URL as malicious.
+      type: String
+    - contextPath: URL.Malicious.Description
+      description: A description of the malicious URL.
+      type: String
+    - contextPath: DBotScore.Indicator
+      description: The indicator that was tested.
+      type: String
+    - contextPath: DBotScore.Type
+      description: The indicator type.
+      type: String
+    - contextPath: DBotScore.Vendor
+      description: The vendor used to calculate the score.
+      type: String
+    - contextPath: DBotScore.Score
+      description: The actual score.
+      type: Number
+    description: Return domain information and reputation
+  - name: cybertotal-ip-whois
+    arguments:
+    - name: ip
+      required: true
+      default: true
+      description: List of IP(s).
+      isArray: true
+    outputs:
+    - contextPath: CyberTotal.WHOIS-IP.scan_date
+      description: 'Scan date format: YYYY-MM-DD HH:mm:SS '
+      type: date
+    - contextPath: CyberTotal.WHOIS-IP.task_id
+      description: The unique id of each scan in CyberTotal.
+      type: string
+    - contextPath: CyberTotal.WHOIS-IP.resource
+      description: The scan target sent to CyberTotal.
+      type: string
+    - contextPath: CyberTotal.WHOIS-IP.message
+      description: Message about this search.
+      type: string
+    - contextPath: CyberTotal.WHOIS-IP.permalink
+      description: The link of this whois report in CyberTotal.
+      type: string
+    - contextPath: CyberTotal.WHOIS-IP.createdAt
+      description: 'Create date format: YYYY-MM-DD HH:mm:SS '
+      type: date
+    - contextPath: CyberTotal.WHOIS-IP.updatedAt
+      description: 'Update date format: YYYY-MM-DD HH:mm:SS '
+      type: date
+    - contextPath: CyberTotal.WHOIS-IP.status
+      description: Status of this IP
+      type: string
+    - contextPath: CyberTotal.WHOIS-IP.domain
+      description: Domain of this IP
+      type: string
+    - contextPath: CyberTotal.WHOIS-IP.domainMd5
+      description: MD5 translation of CyberTotal.WHOIS-IP.domain
+      type: string
+    - contextPath: CyberTotal.WHOIS-IP.domainUnicode
+      description: Encode CyberTotal.WHOIS-IP.domain by using unicode
+      type: string
+    - contextPath: CyberTotal.WHOIS-IP.nameservers
+      description: An array of all DNS nameservers
+      type: string
+    - contextPath: CyberTotal.WHOIS-IP.registrarName
+      description: The name of registrar
+      type: string
+    - contextPath: CyberTotal.WHOIS-IP.registrarEmail
+      description: The email address of registrar
+      type: string
+    - contextPath: CyberTotal.WHOIS-IP.registrarPhone
+      description: The phone number of registrar
+      type: string
+    - contextPath: CyberTotal.WHOIS-IP.registrarCreatedAt
+      description: 'Registrar create date format: YYYY-MM-DD HH:mm:SS '
```

This comes back as epoch time according to the test_data
Either parse it to ISO format, or change documentation
```suggestion
      description: Registrar create date format: ISO 8601
```

---

I'll improve it.